weldx/transformations.py

```diff
@@ -1393,12 +1352,25 @@ def uses_absolute_times(self) -> bool:
 
         Returns
         -------
-        bool :
+        bool :[
```

Comment:
Is the `[` wanted? If yes, what's the purpose?

Reply:
Certainly not 🙃